lesson01/hotel/src/com/andersenhotels/model/Apartment.java

```diff
@@ -0,0 +1,54 @@
+package com.andersenhotels.model;
+
+/**
+ * Represents an apartment in the hotel.
+ */
+public class Apartment {
+    private int id;
+    private double price;
+    private ApartmentStatus status;
+
+    /**
+     * Constructs a new Apartment with the specified ID and price.
+     *
+     * @param id the unique identifier for the apartment.
+     * @param price the price of the apartment.
+     */
+    public Apartment(int id, double price) {
+        this.id = id;
+        this.price = price;
+        this.status = ApartmentStatus.AVAILABLE; // Initially, the apartment is available
+    }
+
+    /**
+     * Gets the ID of the apartment.
+     *
+     * @return the ID of the apartment.
+     */
```

Comment:
these comments are optional.
This is a simple method, so please don't use comments. 
Use it only in complex logic to help colleagues with code)

Reply:
Done